start.sh

```diff
@@ -45,9 +43,9 @@ env_up(){
    KEY_NAME=${KEY_NAME:-cs-key}
 
    aws s3api create-bucket --bucket $S3_BUCKET --region $AWS_REGION --create-bucket-configuration LocationConstraint=$AWS_REGION
-   
-   cd /home/ec2-user/cs-cnap/templates
-   aws s3 cp . s3://${S3_BUCKET} --recursive 
+
```

Comment:
@ryanjpayne Are these scopes still valid? The CNAPP lab instructions only list 4.

Reply:
no, we aren't registering cspm anymore because it often fails when a cloudshare account is already registered with a CID so I think the guide is correct